prism-languages.php

```diff
@@ -45,7 +45,6 @@ function mkaz_code_syntax_block_get_supported_languages() {
 		"toml" => "TOML",
 		"typescript" => "TypeScript",
 		"vim" => "vim",
-		"wasm" => "WebAssembly",
 		"wiki" => "Wiki markup",
 		"xml" => "XML",
 		"yaml" => "YAML",
```

Comment:
:no_entry_sign: **Error**: String &quot;yaml&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).